From f42c5003bb6d755516058e1be86945858f54cca7 Mon Sep 17 00:00:00 2001
From: John Cox <jc@kynesim.co.uk>
Date: Wed, 22 Sep 2021 18:57:19 +0100
Subject: [PATCH] media: rpivid: Remove unused ctx state variable and
 defines

Remove unused ctx state tracking variable and associated defines.
Their presence implies they might be used, but they aren't.

Signed-off-by: John Cox <jc@kynesim.co.uk>
---
 drivers/staging/media/rpivid/rpivid.h | 7 -------
 1 file changed, 7 deletions(-)

--- a/drivers/staging/media/rpivid/rpivid.h
+++ b/drivers/staging/media/rpivid/rpivid.h
@@ -82,12 +82,6 @@ typedef void (*rpivid_irq_callback)(stru
 struct rpivid_q_aux;
 #define RPIVID_AUX_ENT_COUNT VB2_MAX_FRAME
 
-
-#define RPIVID_CTX_STATE_STOPPED	0	/* stream_off */
-#define RPIVID_CTX_STATE_STREAM_ON	1	/* decoding */
-#define RPIVID_CTX_STATE_STREAM_STOP	2	/* in stream_off */
-#define RPIVID_CTX_STATE_STREAM_ERR	3	/* stream_on but broken */
-
 struct rpivid_ctx {
 	struct v4l2_fh			fh;
 	struct rpivid_dev		*dev;
@@ -96,7 +90,6 @@ struct rpivid_ctx {
 	struct v4l2_pix_format_mplane	dst_fmt;
 	int dst_fmt_set;
 
-	atomic_t 			stream_state;
 	struct clk_request		*clk_req;
 	int 				src_stream_on;
 	int 				dst_stream_on;
